Makefile.core.mk

```diff
@@ -474,3 +475,5 @@ include tools/packaging/packaging.mk
 include tests/integration/tests.mk
 
 include common/Makefile.common.mk
+
+include pkg/servicemesh/maistra.mk
```

Comment:
At some point we moved this file to `maistra/maistra.mk`: https://github.com/maistra/istio/blob/maistra-2.3/maistra/maistra.mk

Do you think we could move it now so that next rebases don't need to do it in two steps?

Reply:
Lemme have a look which commit introduced that change.

Reply:
It came with https://github.com/maistra/istio/pull/606. I think we can bring it as a separate PR.